### ISSUE TITLE
Some systems failed to build for Quicklisp dist

### ISSUE THREAD

**Issue by quicklisp** [Lisp-Stat/numerical-utilities#9]
Building with SBCL 2.0.5 / ASDF 3.3.1 for quicklisp dist creation.

Trying to build commit id ca781fccd6cc088b297b6bd9ee85c6510d7cdc48

*num-utils* fails to build with the following error:

```
; caught ERROR:
;   READ error during COMPILE-FILE: Symbol "MAP-ARRAY" not found in the ARRAY-OPERATIONS/ALL package. Line: 268, Column: 25, File-Position: 11224 Stream: #<SB-INT:FORM-TRACKING-STREAM for "file /home/quicklisp/quicklisp-controller/dist/build-cache/numerical-utilities/53d9894ae368388601e6c333b41a7c116750a774/numerical-utilities-20210526-git/src/matrix.lisp" {1006645233}>
...
Unhandled UIOP/LISP-BUILD:COMPILE-FILE-ERROR in thread #<SB-THREAD:THREAD "main thread" RUNNING {1000A18083}>: COMPILE-FILE-ERROR while compiling #<CL-SOURCE-FILE "num-utils" "matrix">

```

*num-utils/tests* fails to build because of a failure in  _num-utils_.

[Full log here](http://report.quicklisp.org/2021-05-26/failure-report/numerical-utilities.html)

**Comment by quicklisp** [comment]
I still get this error.

**Comment by quicklisp** [comment]
I am using the latest commit, 8197d207e5cbd894c68fd488edbc4b8ef8fa4600.

**Comment by Symbolics** [comment]
I can reproduce this. Will fix tomorrow morning.

**Comment by Symbolics** [comment]
Fixed in a0541da

**Comment by quicklisp** [comment]
Thank you, I can confirm that it builds fine for me.
